Core/GDCore/Extensions/Metadata/InstructionMetadata.cpp

```diff
@@ -185,8 +185,18 @@ InstructionMetadata::UseStandardRelationalOperatorParameters(
     size_t operatorParamIndex = parameters.size() - 2;
     size_t valueParamIndex = parameters.size() - 1;
 
+    // Add after calculating the operatorParamIndex and valueParamIndex
+    if (type == "string") {
+      AddParameter("yesorno", _("Case insensitive"));
+    }
+
+    gd::String templateSentence = "";
     if (isObjectInstruction || isBehaviorInstruction) {
-      gd::String templateSentence = _("<subject> of _PARAM0_ <operator> <value>");
+      if (type == "string") {
+        templateSentence = _("<subject> of _PARAM0_ <operator> <value>, Case insensitive: _PARAM3_");

```

Comment:
Is the negative form necessary to ensure the compatibility?

Also, I guess parenthesis would be better.
```suggestion
        templateSentence = _("<subject> of _PARAM0_ <operator> <value> (case insensitive: _PARAM3_)");
```

Reply:
I didn't understand, what did you mean by the "negative form", you meant the if-else?

Reply:
I'm now studying if we can catch a "null" value or undefined for the new parameter.

Maybe if _NULL_ means is a old condition / old project and that means "case sensitive"
And if true / false is a condition made with the new feature.

If it works we can know the situation and I could change the "insensitive" for "sensitive".